src/fittable_models.jl

```diff
@@ -0,0 +1,45 @@
+# Convenience function for wavelength conversion
+aa_to_invum(wave::Real) = 10000 / wave
+aa_to_invum(wave::Quantity) = aa_to_invum(ustrip(u"angstrom", wave))
+
+"""
+    FM90(C1 = 0.10, C2 = 0.70, C3 = 3.23, C4 = 0.41, xo = 4.60, γ = 0.9)(λ::Real)
+
+The parameters C1, C2, C3, C4, xo and γ are y-intercept, slope, bump amplitude, FUV rise amplitude,
```

Comment:
This is not a grammatically correct sentence.   Perhaps create a list instead
```
### Parameters
* `c1` - y-intercept
```
Etc. 

Are these the exact variable names used from the paper? They’re awfully not indicative of what they do. We also may want to set this up using Parameters.jl for kwdef the struct. What do you think @giordano ?

Reply:
Yes, I was thinking about using `Base.@kwdef`, but if this is going to be a parametric type I think that this macro supports them only in Julia v1.1+ (better double check this), so `Parameters.jl` would be a better option in that case.

Reply:
I referred to astropy codebase. :sweat_smile: I am switching to the list format.
Sorry about the grammatical mistake.

Reply:
Don’t sweat the grammar! If you can find the parameter names from the paper they might have better names, or perhaps worse. I just want to minimize the _need_ for documentation to understand what those values are.

Reply:
Also, base kwdef doesn’t exist in 1.0, so we should use Parameters.jl. They also have a neat syntax for parameteized and default values struct, just like this!

Reply:
> base kwdef doesn’t exist in 1.0

I think it does, but it doesn't support parametric structs, as I said above:

```
help?> Base.@kwdef
  @kwdef typedef

  This is a helper macro that automatically defines a keyword-based constructor for the type declared in the expression typedef, which must
  be a struct or mutable struct expression. The default argument is supplied by declaring fields of the form field::T = default or field =
  default. If no default is provided then the keyword argument becomes a required keyword argument in the resulting type constructor.

  Inner constructors can still be defined, but at least one should accept arguments in the same form as the default inner constructor (i.e.
  one positional argument per field) in order to function correctly with the keyword outer constructor.

  │ Julia 1.1
  │
  │  Base.@kwdef for parametric structs, and structs with supertypes requires at least Julia 1.1.
```